app/services/replication/aws_provider.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module S3
+module Replication
```

Comment:
just spitballing here:  S3 has some meaning, since everything we use is built around it.  Replication has better meaning ... but should we drop S3?   Is there a utility in S3Base class name?